backend-diagram-converter/cli/README.md

```diff
@@ -1,23 +1,131 @@
 # Conversion CLI tool
 
-## Features
+The command line interface tool can convert a Camunda Platform 7 diagram or all diagrams inside a folder (plus sub directories)
+or convert all diagrams from a Camunda Platform 7 process engine.
 
-### Migration of existing diagrams
+All diagrams that should be converted need to have the `.bpmn` or `.bpmn20.xml` file ending.
 
-The CLI tool can migrate a given diagram or all content inside a folder (plus subdirectories).
+The tool logs all results (either location of the new file or the exception the migration tool faced during migration).
+Diagrams that are already in C8 format will be omitted but logged. So you can follow the progress of converting.
 
-All diagrams that should be converted need to have the `.bpmn` or `.bpmn20.xml` file ending.
+In case you run the CLI tool on your Windows computer and your process diagram contains Umlaute,
+add the java option `-Dfile.encoding=UTF-8` between the java and -jar parameters:
+
+```
+java -Dfile.encoding=UTF-8 -jar backend-diagram-converter-cli-version.jar
+```
+
+## Commands
+
+### Convert diagrams from the local file systems
+
+```
+Usage: backend-diagram-converter-cli local [-dhoV] [--check] [--csv] [-nr]
```

Comment:
Would we really like to document what the CLI help would print out? This can be a lot of effort for something that can be looked up easily.

Reply:
It's easier than writing a summary.

People may forget about the -h option.

I like to read about the options before I install a tool. 

That's what man pages do.

Do you have another proposal, how to list all available commands, parameters and options in the README?

In the end, I just copied it from the command line and pasted it into the markdown.

Reply:
I see, I just fear that docs will nit be aligned with the tool in the future.

I will add a task to figure this out.